server/src/entity/Visit.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  Entity,
+  JoinTable,
+  ManyToMany,
+  ManyToOne,
+  OneToMany,
+  PrimaryGeneratedColumn,
+  OneToOne,
+} from 'typeorm';
+
+import { Account } from './Account';
+import { Shift } from './Shift';
+import { Transaction } from './Transaction';
+
+export enum paymentMethod {
+  SPLIT = 'split',
+  ALTOGETHER = 'altogether',
+}
+
+@Entity()
+export class Visit extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  public id: string;
+
+  @ManyToMany((type) => Account, (account) => account.visits)
+  public users: Account[];
+
+  // TODO: insert orders here
+
+  @OneToMany((type) => Shift, (shift) => shift.visits)
+  public assignee: Shift;
```

Comment:
This assumes that only one server can be associated with a table. This is probably a fair assumption for this part of the project; however, I would document this so that we can look at it later.

Reply:
I had it documented under #20, but I can create a new issue if you like.

Reply:
You should probably create a new issue. We will most likely close #20 after this pr.

Reply:
See #41